presto-main/src/test/java/com/facebook/presto/operator/TestClassificationMissRateAggregation.java

```diff
@@ -34,7 +34,7 @@ public TestClassificationMissRateAggregation()
         final ArrayList<Double> expected = new ArrayList<>();
         while (iterator.hasNext()) {
             final BucketResult result = iterator.next();
-            final double missRate = (result.totalFalseWeight - result.remainingFalseWeight) / result.totalTrueWeight;
+            final double missRate = result.remainingFalseWeight / result.totalTrueWeight;
```

Comment:
Maybe also making `remainingFalseWeight`, etc in `BucketResult` private and provide getters?

Reply:
Did that.